packages/m/miaoyan.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "miaoyan",
+  "description": "Lightweight Markdown app to help you write great sentences.",
+  "filename": "package.json",
```

Comment:
You are only including js/css/ttf files -- `filename` should be the name of one of the files you are including, the base file that users should import into their site to use your project